packages/app/src/SearchInputV2.tsx

```diff
@@ -1,11 +1,27 @@
-import { useEffect, useMemo, useRef, useState } from 'react';
+import { useEffect, useRef, useState } from 'react';
 import { useController, UseControllerProps } from 'react-hook-form';
 import { useHotkeys } from 'react-hotkeys-hook';
-import { TableConnection } from '@hyperdx/common-utils/dist/metadata';
+import { Field, TableConnection } from '@hyperdx/common-utils/dist/metadata';
 import { genEnglishExplanation } from '@hyperdx/common-utils/dist/queryParser';
 
 import AutocompleteInput from '@/AutocompleteInput';
-import { useAllFields } from '@/hooks/useMetadata';
+
+import {
+  ILanguageFormatter,
+  useAutoCompleteOptions,
+} from './hooks/useAutoCompleteOptions';
+
+export class LuceneLanguageFormatter implements ILanguageFormatter {
+  formatFieldValue(f: Field): string {
+    return f.path.join('.');
```

Comment:
no need to fix it for now but I'm curious what happened if the key has '.' in it